pygenometracks/tracks/BedGraphTrack.py

```diff
@@ -216,6 +222,7 @@ def plot(self, ax, chrom_region, start_region, end_region):
             score_list, x_values = self.get_values_as_bdg(score_list,
                                                           pos_list)
 
+        # I do not see how this part of the code can be executed:
```

Comment:
Should we remove this part?

Reply:
4C?  No, I am afraid it was assigned there to plot matrices of 4c.

Reply:
But how is it possible to fill these conditions:
```
if 'extra' in self.properties and self.properties['extra'][0] == '4C':
```
`self.properties['extra']` should be a string, then `self.properties['extra'][0]` will be only one character...

There is no example, showing how to use it.

Reply:
I have also checked and found no example. I wonder if there was a difference between the dict we have now and the one which was around when it was implemented. I see your point, just try to figure out if there is no other place in the code which depends on it. To me also it can be removed, i agree with you. Just want to be sure that we dont break anything. Have you already tested for that?

Reply:
I removed it into a separated branch and all tests worked fine.

Reply:
Let's say if its not tested remove it :)
We can add it later if we figure out what it was :)

Reply:
OK done